apps/web/test/lib/getSchedule.test.ts

```diff
@@ -27,57 +34,117 @@ declare global {
 }
 
 expect.extend({
-  toHaveTimeSlots(schedule, expectedSlots: string[], { dateString }: { dateString: string }) {
-    expect(schedule.slots[`${dateString}`]).toBeDefined();
-    expect(schedule.slots[`${dateString}`].map((slot: { time: string }) => slot.time)).toEqual(
-      expectedSlots.map((slotTime) => `${dateString}T${slotTime}`)
-    );
+  toHaveTimeSlots(
+    schedule: { slots: Record<string, Slot[]> },
+    expectedSlots: string[],
+    { dateString }: { dateString: string }
+  ) {
+    if (!schedule.slots[`${dateString}`]) {
```

Comment:
don't use expect for assertions in custom expectations. This is to avoid wrong line from being reported as failure